proposals/proposal-template.md

```diff
@@ -0,0 +1,112 @@
+<!---
+Welcome! Thank you for contributing to the U.S. Web Design System.
+Your contributions are vital to our success and we are glad you want to share your ideas.
+
+You can find full instructions for creating a proposal in the uswds-proposals repo:
+https://github.com/amyleadem/uswds-proposals/edit/add-readme-and-template/README.md#submitting-a-new-proposal
+
+More information about contributing to USWDS can be found on the contribution page:
+https://designsystem.digital.gov/about/contribute/
+ -->
+
+## Summary*
+
+_Provide a short paragraph or bulleted list that gives an overview of the proposed component._
+
+## Motivation*
+
+_Explain why we should include this component in USWDS._
+
+- _What user need does this component address?_
+- _What problem does this component solve?_
+
+## Proposed solution*
+
+_Thoroughly describe the proposed component. Use the following sections to add detail:_
+
+### Functionality*
+
+_Describe the component's core functionality. If the component is dynamic, describe its interactive states._
+
+### Visuals*
+
+_Add diagrams, images, and/or wireframes that visually demonstrate this component._
+
+### Variants*
+
+_Describe any variants planned for this component._
+
+### Settings*
+
+_Describe which component elements should be customizable with theme settings._
+
+### Accessibility*
+
+_Identify any accessibility considerations._
+
+- _Will the component be compatible with assistive technologies?_
+<!-- 
+Consider the following:
+- Keyboard-only interaction
+- Screen readers
+- Voice command
+- High-contrast mode
+- Zoom magnification 
+-->
+- _Are there any audiences that might struggle with this component?_
+<!--Explain how this component will accommodate these needs.-->
+
+### Implementation plan*
+_Add details about how this plan should be implemented._
+
+- _Will the component be implemented in phases?_
+- _Are there any potential blockers?_
+- _Are there any dependencies?_
```

Comment:
I like these suggestions. I have added them as comments in the markdown for now so that we don't lose track. I'd like to discuss how much we want to show in the preview mode for the template vs what we might want to add as a comment in the markdown. 

I like comments because it feels easier to read and less intimidating, but I don't want good notes to be hidden if people aren't looking at the markdown file. 

Going to leave this comment unresolved until we decide on a strategy.